josm-presets/pl-signals.xml

```diff
@@ -0,0 +1,717 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<presets baselanguage="pl" xmlns="http://josm.openstreetmap.de/tagging-preset-1.0" author="Grzegorz Szymaszek" shortdescription="OpenRailwayMap PL: sygnalizacja" version="2018-08-23">
+	<chunk id="signal_ref">
+		<key key="railway" value="signal"/>
+		<text key="ref" text="Identyfikator sygnalizatora/wskaźnika"/>
+	</chunk>
+	<chunk id="signal_directions">
+		<combo key="railway:signal:direction" display_values="do przodu,do tyłu" text="Kierunek" values="forward,backward"/>
+	</chunk>
+	<chunk id="signal_directions_all">
+		<combo key="railway:signal:direction" display_values="do przodu,do tyłu,w obu kierunkach" text="Kierunek" values="forward,backward,both"/>
+	</chunk>
+	<chunk id="signal_position">
+		<combo default="right" display_values="po lewej,po prawej,na kratownicy lub podobnej konstrukcji ponad torem" key="railway:signal:position" text="Umieszczenie względem toru" values="left,right,bridge"/>
+	</chunk>
+	<group icon="signals.png" name="OpenRailwayMap PL: sygnalizacja">
+		<group name="Semafory">
+			<item name="Semafor kształtowy jednoramienny" type="node">
+				<key key="railway:signal:main" value="PL-PKP:sr"/>
+				<key key="railway:signal:main:form" value="semaphore"/>
+				<key key="railway:signal:main:states" value="PL-PKP:sr1;PL-PKP:sr2"/>
+				<reference ref="signal_ref"/>
+				<check key="railway:signal:main:deactivated" text="Wyłączony z użytku"/>
+				<reference ref="signal_position"/>
+				<reference ref="signal_directions"/>
+				<space/>
+				<combo display_values="normalny" key="railway:signal:main:height" text="Wysokość [m]" values="normal"/>
```

Comment:
Do these signals only exist in one height? If yes, is a combobox really the right input for that? Other options would be e.g. key (i.e. always write that when the preset is used) or maybe a checkbox. Maybe also move this into a chunkß

Reply:
AFAIK, Polish mechanical semaphores either have “normal” height or are mounted above tracks. I don’t know if they’re always of the same exact height, so I’ve used a combobox to allow specifying exact height if it’s known. What do you think about replacing it with
```xml
<key key="railway:signal:main:height" match="key" value="normal"/>
```
?